ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -16,16 +16,42 @@
 open Datamodel_types
 open CommonFunctions
 module Types = Datamodel_utils.Types
+module StringSet = Set.Make (String)
 
 let templates_dir = "templates"
 
 let ( // ) = Filename.concat
 
+let acronyms =
+  [
+    "id"
+  ; "ip"
+  ; "vm"
+  ; "api"
+  ; "uuid"
+  ; "cpu"
+  ; "tls"
+  ; "https"
+  ; "url"
+  ; "db"
+  ; "xml"
+  ; "eof"
+  ]
+  |> StringSet.of_list
+
+let is_acronym word = StringSet.mem word acronyms
+
 let snake_to_camel (s : string) : string =
   Astring.String.cuts ~sep:"_" s
   |> List.map (fun s -> Astring.String.cuts ~sep:"-" s)
   |> List.concat
```

Comment:
Maybe carry over changes from the other PR? `List.concat_map (Astring.String.cuts ~sep:"-")` here as before. Ignore me if you have a reconciliation plan already.

Reply:
Yeah, sorry.